afs/media/js/views/components/workflows/analysis-areas-annotation-step.js

```diff
@@ -162,8 +163,9 @@ define([
             physicalThingAnnotationNode.active(true); 
         };
 
-        this.saveAnalysisAreaTile = function() {
-            self.tile.save().then(function(data) {
+        this.saveAnalysisAreaTile = function () {
+            self.tile.save().then(function (data) {
```

Comment:
nit throughout: can the spacing between `function` and arg be removed?

Reply:
Yes - it's also good to note that in my next PR, some of this is auto-formatted.  Perhaps some common style guide/formatter might be in order (prettier?)